src/ui/issuecolumn/IssueColumn.java

```diff
@@ -68,6 +70,21 @@ public IssueColumn(UI ui, Stage mainStage, Model model, ColumnControl parentColu
 		this.ui = ui;
 		getChildren().add(createFilterBox());
 		setupIssueColumnDragEvents(model, columnIndex);
+		this.setOnMouseClicked(e-> {
+			ui.triggerEvent(new ColumnClickedEvent(columnIndex));
+			getParent().requestFocus();
+		});
+		focusedProperty().addListener(new ChangeListener<Boolean>() {
+			@Override
+			public void changed(ObservableValue<? extends Boolean> unused, Boolean wasFocused, Boolean isFocused) {
+				if (!isFocused) {
+					if (getStyleClass().contains("issue-panel-focused"))
+						getStyleClass().remove("issue-panel-focused");
+					return;
+				}
+				getStyleClass().add("issue-panel-focused");
+			}
+		});
 	}
```

Comment:
Is there a need to check for the presence of the style class, i.e. could the following work?

```java
if (isFocused) {
    getStyleClass().add("issue-panel-focused");
} else {
    getStyleClass().remove("issue-panel-focused");
}
```